lib/stripe/account.rb

```diff
@@ -1,9 +1,27 @@
 module Stripe
-  class Account < SingletonAPIResource
+  class Account < APIResource
+    include Stripe::APIOperations::Create
+    include Stripe::APIOperations::List
+    include Stripe::APIOperations::Update
+
+    def url
+      if self['id']
+        super
+      else
+        "/v1/account"
+      end
+    end
+
+    # @override To make id optional
+    def self.retrieve(id=nil, opts={})
+      super
+    end
+
     def deauthorize(client_id, opts={})
-      api_key, headers = Util.parse_opts(opts)
-      response, api_key = Stripe.request(:post, '/oauth/deauthorize', api_key, { 'client_id' => client_id, 'stripe_user_id' => self.id }, headers, Stripe.connect_base)
-      Util.convert_to_stripe_object(response, api_key)
+      opts = {:api_base => Stripe.connect_base}.merge(Util.normalize_opts(opts))
+      response, opts = request(:post, '/oauth/deauthorize', { 'client_id' => client_id, 'stripe_user_id' => self.id }, opts)
```

Comment:
Yea, this was a PR from an external contributor. I was a bit iffy on adding direct connect API support to the bindings, but in the end it seemed worth it. stripe-ruby is the only one with this right now.